src/nwbinspector/checks/behavior/__init__.py

```diff
@@ -0,0 +1,13 @@
+import warnings
+
+message = (
+    "All submodules below the level 'nwbinspector.checks'  have been deprecated. "
+    "Please retrieve the specific check function(s) you wish to use from the `available_checks` registry "
```

Comment:
Should the `default_check_registry` be recommended  in this case instead of the `available_checks` list since a user would likely want to retrieve the specific check function(s) with the check name?

Reply:
That is an interesting question, keep it in mind going forward

I have other ideas for how to expose listings of checks that are distinct from the raw registry (since interactions with that underly the source of multiple issues), which I might even swap to be privately managed and not exposed